spec/models/policy_spec.rb

```diff
@@ -75,6 +75,9 @@
     end
 
     context 'on update' do
+      before { Sidekiq::Testing.inline! }
+      after  { Sidekiq::Testing.fake! }
```

Comment:
Not sure what changes internally for Active Job to all of a sudden require this, but w/e.

Reply:
@ezekg I wonder if this is related to rails/rails#48585

Reply:
@zzak that's it! Makes sense now that I think about it. Thanks.